pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -74,16 +74,19 @@ def panic_output_decode_script(self) -> t.Optional[str]:
         Returns:
             Panic output decode script path
         """
-        script_filepath = self._panic_output_decode_script or os.path.join(
-            os.getenv('IDF_PATH', 'IDF_PATH'),
-            'tools',
-            'gdb_panic_server.py',
-        )
-        if not os.path.isfile(script_filepath):
-            raise ValueError(
-                'Panic output decode script not found. Please use --panic-output-decode-script flag '
-                'to provide script or set IDF_PATH (Default: $IDF_PATH/tools/gdb_panic_server.py)'
-            )
+        import importlib.util
+
+        script_filepath = self._panic_output_decode_script
+        if not script_filepath or not os.path.isfile(script_filepath):
+            mdl = importlib.util.find_spec('esp_idf_panic_decoder.gdb_panic_server')
+            if not mdl:
```

Comment:
IMO we should add this dependency `esp-idf-panic-decoder` to `pytest_embedded_idf/pyproject.toml`, since it's a dependency of ESP-IDF project [ref](https://github.com/espressif/esp-idf/blob/master/tools/requirements/requirements.core.txt#L18)

and btw, don't use acronyms frequently. `module` is not a keyword, nor a long word :)

Reply:
Added to dep list